src/ErrorBoundary.ts

```diff
@@ -112,7 +112,7 @@ export class ErrorBoundary extends Component<
         },
       },
       childToRender
-    ) as ReactElement;
+    );
```

Comment:
Note that the change you're removing here was added to address issue #133 so I don't know of landing this change would regress that one.

Reply:
_"it worked on my machine"_

Reply:
You should be able to run `pnpm prerelease` locally and verify the output fwiw.

Reply:
I did that. Posted the output above.

I wanted to make sure it's the same for you too.

Reply:
Oh okay gotcha. I'll try to check this out sometime this afternoon. 👍🏼